### DISCUSSION TITLE
Is there a way to specify only the latest part of the url in href links?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#4217] [Q&A]
Let's say I'm in `players/<PLAYER_ID>/settings/__layout.svelte` and I need two more nested pages:

1. `/profile` in `/players/[PLAYER_ID]/settings/profile.svelte`
1. `/notifications` in `/players/[PLAYER_ID]/settings/notifications.svelte`.

In `__layout.svelte` I have the links:

```svelte
<a href={`/players/${player.id}settings/profile`}>Profile</a>

<a href={`/players/${player.id}settings/notifications`}>Notifications</a>
```

Is there a way to avoid all the links in that `href`?

Something like `/notifications` only?

**Comment by mustofa-id** [comment]
There is no way.
Why would you need this?

**Reply by frederikhors** [reply]
To not repeat the link every time in the page or to avoid a variable. I think goto or something else can help here.

**Reply by mustofa-id** [reply]
What if there is also `/notifications` route in your root and you need to go to there from settings page?

here is a tip if you don't want to repeat the link and avoid a variable:
```svelte
{#each [['/profile', 'Profile'], ['/notifications', 'Notifications']] as [path, label]}
    <a href="/players/{player.id}/settings/{path}">{label}</a>
{/each}
```
:)

**Reply by frederikhors** [reply]
Yeah. Thank you.

**Reply by mustofa-id** [reply]
Anytime.
Please consider to mark the question as answered if this help you.

**Reply by mustofa-id** [reply]
I missed the question, using `href="notifications"` should be work.

**Comment by diego-yason** [comment]
You could use a [relative href](https://stackoverflow.com/a/24028813/15052894)

[Here is an example](https://stackblitz.com/edit/sveltejs-kit-template-default-aug4cd?file=src/routes/user/[id]/bar.svelte)

Warning: If the ID changes for some reason, it won't be reflected until the dynamic part of the URL is updated
